src/index.html

```diff
@@ -17,6 +17,10 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch stopwatch--speed-up">
```

Comment:
The BEM modifier `stopwatch--speed-up` suggests that there may be a variation in the speed of the stopwatch. Ensure that this modifier is used correctly in the styles and that there's a base class `stopwatch` that defines the necessary properties before the modifier is applied.